client/app/visualizations/map/index.js

```diff
@@ -229,7 +247,21 @@ const MapRenderer = {
           const groupColor = this.options.groups[name].color;
           marker = heatpoint(lat, lon, groupColor);
         } else {
-          marker = createMarker(lat, lon);
+          if (this.options.customizeMarkers) {
+            const icon = L.BeautifyIcon.icon({
+              iconShape: this.options.iconShape,
+              icon: this.options.iconFont,
+              iconSize: this.options.iconShape === 'rectangle' ? [22, 22] : false,
```

Comment:
@deecay can you elaborate on why rectangle is a special case?

Reply:
Also, it seems `[22,22]` is the BeautifyMarker default so perhaps `this.options.iconShape === 'rectangle' ? null : false` could be better?

Reply:
`[22, 22]` is there to make the marker a square. Comparison:
![image](https://user-images.githubusercontent.com/1698635/59556750-ebaf9d80-9002-11e9-93b8-4e2352da11ab.png)

Another example at the PR comment above, with ambulance icon.

I thought square looked better and also consistent with other marker types.

Reply:
Strangely, I couldn't make the default value work, so I had to specify the size.

Reply:
Seems like default of `[22, 22]` only applies to circles.

https://github.com/marslan390/BeautifyMarker/blob/b650118f45a683437818278d1df7cfa778a9e9d8/leaflet-beautify-marker-icon.js#L48